### DISCUSSION TITLE
[Core Protocol] Testing Op-challenger in devnet

### DISCUSSION THREAD

**Discussion by priyanshuthapliyal55** [ethereum-optimism/developers#362] [Core Protocol] [answered]
### Did you check the documentation?

- [X] Yes

### Did you read the specs?

- [X] Yes
- [ ] No

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I have deployed OP-CHALLEGER on devnet for testing it now how can i test it as in readme it is not clear about how we can test it ?

**Comment by hash1go** [comment]
Hello. To test the op-challenger, please construct a devnet including other components.
https://docs.optimism.io/chain/testing/dev-node

**Reply by priyanshuthapliyal55** [reply]
I have deployed devnet it is running even op-challeger is showing challeger service started , I am running op-challenger using binari instead of docker compose. Now i want to test op-challenger how can i do that ?

**Reply by hash1go** [reply, marked as answer]
There isn’t a specific guide for testing op-challenger. Since op-challenger operates with fault proofs, a good way to test op-challenger would be by testing the fault proofs themselves.
